src/config.rs

```diff
@@ -325,6 +325,16 @@ impl Config {
             .unwrap_or(35 * 1024 * 1024))
     }
 
+    pub fn get_max_retry_count(&self) -> Result<u32, Error> {
+        if env::var_os("SENTRY_REQUEST_MAX_RETRY").is_some() {
+            Ok(env::var("SENTRY_REQUEST_MAX_RETRY")?.parse()?)
+        } else if let Some(val) = self.ini.get_from(Some("http"), "max_retry_count") {
```

Comment:
I know that we've been inconsistent with these, but can we make them more alike? For example `export SENTRY_HTTP_UPLOAD_RETRIES=2` and `http.upload_retries = 2`.

On a side note, after releasing this we should also document the config [here](https://docs.sentry.io/cli/configuration/#configuration-values)

Reply:
Updated!